src/sql/src/plan/statement/dml.rs

```diff
@@ -1210,11 +1210,18 @@ fn plan_copy_from(
         bail_unsupported!("COPY FROM ... WITH (FILES ...) only supported from a URL")
     }
 
-    let (id, _, columns) = query::plan_copy_from(scx, table_name, columns)?;
+    let (id, source_desc, columns, maybe_mfp) = query::plan_copy_from(scx, table_name, columns)?;
+
+    let Some(mfp) = maybe_mfp else {
+        bail_unsupported!("COPY FROM ... expects an MFP to be produced");
```

Comment:
This will produce an error string like so: "COPY FROM ... expects an MFP to be produced not yet supported", which is probably confusing. Is this something users can trigger with syntax? If not what we do in other places in the planner is something like `sql_bail!("[internal error]: COPY FROM ... expects an MFP to be produced")`

Reply:
Users should not be able to trigger this syntax, it's solely a programming error. Changed the returned error to be what you suggested!